UnityProject/Assets/Scripts/Chat/Chat.Process.cs

```diff
@@ -87,19 +87,19 @@ private static (string, ChatModifier) ProcessMessage(ConnectedPlayer sentByPlaye
 		// Emote
 		if (message.StartsWith("*") || message.StartsWith("/me ",true,CultureInfo.CurrentCulture))
```

Comment:
imo these checks should be against `message.ToLower()` since someone could write `/Me` and these would fail

Reply:
No? The second argument of startswith is an ignorecase flag

Reply:
oh right.. that's not obvious unless IDE gives you a hint